src/langsmith/composite-evaluators.mdx

```diff
@@ -0,0 +1,219 @@
+# Skip rows with any missing metric
+required = list(DEFAULT_WEIGHTS.keys())
+missing_cols = [c for c in required if c not in df.columns]
+if missing_cols:
+    raise ValueError(f"Missing columns in DataFrame: {missing_cols}")
+
+# Calculate weighted score. This can be any aggregation function
+def row_score(row: pd.Series) -> float:
+    if row[required].isna().any():
+        return float("nan")
+    return float(sum(row[c] * DEFAULT_WEIGHTS[c] for c in required))
+
+df[WEIGHTED_KEY] = df.apply(row_score, axis=1)
+
+# Write feedback back to LangSmith
+for _, row in df.iterrows():
+    run_id = row["id"]
+    score = row[WEIGHTED_KEY]
+    if isinstance(score, (int, float)) and not math.isnan(score):
+        client.create_feedback(
+            run_id=run_id,
+            key=WEIGHTED_KEY,
+            score=float(score)
+        )
+```
+</Accordion>
+
+
+## How to create a composite evaluator using the UI
+
+Composite evaluators can be created on either a tracing project (for online evaluations) or a dataset (for offline evaluations). With composite evaluators in the UI, you can compute a weighted average or weighted sum of multiple evaluator scores, with configurable weights.
+
+![Composite evaluator](/langsmith/images/create_composite_evaluator.png)
+
+### 1. Navigate to the tracing project or dataset
+
+To start configuring a composite evaluator, navigate to the **Tracing Projects** or **Dataset & Experiments** tab and select a project or dataset.
+- From within a tracing project: **+ New** > **Evaluator** > **Composite score**
+- From within a dataset: **+ Evaluator** > **Composite score**
+
+### 2. Configure the composite evaluator
+
+- Name your evaluator
+- Select an aggreagation method, either **Average** or **Sum**.
+    - **Average**: ∑(weight*score) / ∑(weight).
+    - **Sum**: ∑(weight*score).
+- Add the feedback keys you want to include in the composite score.
+- Add the weights for the feedback keys. By default, the weights are equal for each feedback key. Adjust the weights to increase or decrease the importance of specific feedback keys in the final score.
+- Click **Create** to save the evaluator.
+
+<Tip> If you need to adjust the weights for the composite scores, they can be updated after the evaluator is created. The resulting scores will be updated for all runs that have the evaluator configured. </Tip>
+
+### 3. View composite evaluator results
+
+Composite scores are attached to a run as **feedback**, similar feedback from a single evaluator. How you can view them depends on where the evaluation was run:
+
+**On a tracing project**:
+- Composite scores appear as feedback on runs.
+- [Filter for runs](/langsmith/filter-traces-in-application) with a composite score, or where the composite score meets a certain threshold.
+- [Create a chart](/langsmith/dashboards#custom-dashboards) to visualize trends in the composite score over time.
+
+**On a dataset**:
+- View the composite evaluator results in the **Experiments** tab.
```

Comment:
maybe worth calling out the filtering/sorting support for composite scores in this view? e.g. "You can filter experiments based on the average composite score of their runs"